### PR TITLE
Add auto test for nVeto

### PR DESCRIPTION
**Pull request by mzks**
**What is the problem / what does the code in this PR do**

**WFSim needs you:**
 - Please add a test for this PR, as a bare minimum, make sure it's covered in coveralls!
 - Can you add a docsting to all your functions?

### CODE REVIEW

File: wfsim/load_resource.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def __init__(self, config=None):
         elif config['detector'] == "XENON1T":
             url_base = f'https://raw.githubusercontent.com/XENONnT/strax_auxiliary_files/{commit}/sim_files'
         elif config['detector'] == "XENONnT":
-            url_base = f'https://raw.githubusercontent.com/XENONnT/WFSim/{commit}/files'
+            url_base = f'https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/{commit}/sim_files'
```
Comment (jorana) [inline wfsim/load_resource.py:79]: @mzks , this will never work as it is private

File: wfsim/strax_interface.py [hunk 2]
```diff
@@ -359,7 +342,7 @@ def __init__(self, config):
         log.debug(f'Raw data is set')
         self.record_buffer = np.zeros(5000000,
                                       dtype=strax.raw_record_dtype(samples_per_record=strax.DEFAULT_RECORD_LENGTH)) # 2*250 ms buffer
-        self.truth_buffer = np.zeros(10000, dtype=instruction_dtype + truth_extra_dtype + [('fill', bool)])
+        self.truth_buffer = np.zeros(100000, dtype=instruction_dtype + truth_extra_dtype + [('fill', bool)])
```
Comment (jorana) [inline wfsim/strax_interface.py:345]: I think I have seen too that we need this

**Review by jorana — COMMENTED** [review]
Thanks Keita, always good to add tests for previously untested code. Briefly looking at the code I just spotted one issue:

File: wfsim/strax_interface.py (hunk 2 above)
Comment (ramirezdiego) [reply wfsim/strax_interface.py:345]: Hey @jorana, please notice that you're reviewing changes from #125. Keita commited his changes of top of the branch being reviewed in there.

Comment (jorana) [reply wfsim/strax_interface.py:345]: Thanks Diego, I wondered as such as some of the lines changed looked a little out of place given the scope of this PR, thanks for clarifying 😄

File: wfsim/load_resource.py (hunk 1 above)
Comment (mzks) [reply wfsim/load_resource.py:79]: I just copied TPC one. How should I modify them?

Comment (jorana) [reply wfsim/load_resource.py:79]: I think `url_base = f'https://raw.githubusercontent.com/XENONnT/WFSim/{commit}/files'` (as it was) is of limited use but more useful than the `f'https://raw.githubusercontent.com/XENONnT/private_nt_aux_files/{commit}/sim_files'` which certainly won't work.

**Comment by petergaemers** [comment]
@mzks Sorry for ignoring this so far. Since this is marked as a draft what is still missing?
